src/Protocol/Http/Authenticator/JwtAuthenticator.php

```diff
@@ -30,24 +30,61 @@ class JwtAuthenticator implements AuthenticatorInterface
      */
     private $jwt;
 
+    /**
+     * @var string
+     */
+    private $jws = '';
+
+    /**
+     * @var \DateInterval
+     */
+    private $jwsLifetime = null;
+
+    /**
+     * @var \DateTime
+     */
+    private $jwsValidTo = null;
+
     /**
      * Constructor.
      *
-     * @param JwtInterface $jwt
+     * @param JwtInterface  $jwt
+     * @param \DateInterval $jwsLifetime
      */
-    public function __construct(JwtInterface $jwt)
+    public function __construct(JwtInterface $jwt, \DateInterval $jwsLifetime = null)
     {
         $this->jwt = $jwt;
+
+        $this->jwsLifetime = is_null($jwsLifetime) ? new \DateInterval('P1H') : $jwsLifetime;
```

Comment:
Please do not set the default value to lifetime.

Problem: How I can regenerate token for each request? 
Maybe the developer should regenerate the token by he business logic.